README.md

```diff
@@ -462,13 +460,64 @@ Additionally, the OIDC `redirect_uri` parameter must be set to the HTTP API's au
 This is required, as the Authorization Code Grant needs to be redirected back to the HTTP API, to complete
 HTTP API token generation.
 
-### Add your specified `irods_user_claim` to the user's claims
+### Configure your 'user_mapping'
```

Comment:
This section speaks of developing your own plugin, but we don't provide any information about how to do that.

We need to add a third section which helps developers interested in this route.

Reply:
Added a section on plugin development.